README.md

```diff
@@ -4,9 +4,9 @@ https://github.com/rjl20/hpmor
 Maintainer: Reuben Thomas <rrt@sc3d.org>
 
 A LaTeX version of [the popular didactic fan-fiction](http://www.hpmor.com)
-by Eliezer Yudkowsky, which can make a PDF e-book (one file) or printable
-books (either one or six volumes; the latter option is more practical to
-bind). There are also dust jackets for the printable volumes.
+by Eliezer Yudkowsky, which can make e-books in PDF, ePub and Mobi formats,
+and six PDF volumes that can be printed and bound.
+There are also dust jackets for the printable volumes.
```

Comment:
Sorry to be a pedant: this line needs wrapping on to the previous.

Reply:
Sorry, I don't get it. May I ask you to do the remaining modification yourself? I guess that is faster than ping-pong ;-)